src/Symfony/Component/AssetMapper/Command/ImportMapOutdatedCommand.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\Command;
+
+use Symfony\Component\AssetMapper\ImportMap\ImportMapUpdateChecker;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(name: 'importmap:outdated', description: 'Lists outdated JavaScript packages and their latest versions')]
+final class ImportMapOutdatedCommand extends Command
+{
+    private const COLOR_MAPPING = [
+        'update-possible' => 'yellow',
+        'semver-safe-update' => 'red',
+        'up-to-date' => 'green',
+    ];
+
+    public function __construct(
+        protected readonly ImportMapUpdateChecker $updateChecker,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument(
+                name: 'packages',
+                mode: InputArgument::IS_ARRAY | InputArgument::OPTIONAL,
+                description: 'List packages\' names',
+            )
+            ->addOption(
+                name: 'format',
+                mode: InputOption::VALUE_REQUIRED,
+                description: sprintf('The output format ("%s")', implode(', ', $this->getAvailableFormatOptions())),
+                default: 'txt',
+            )
+            ->addOption(
+                name: 'all',
+                shortcut: 'a',
+                mode: InputOption::VALUE_NONE,
+                description: 'Show all packages, not just outdated ones.',
+            )
+            ->addOption(
+                name: 'ignore',
+                mode: InputOption::VALUE_REQUIRED,
+                description: 'The package(s) to ignore',
+            )
+            ->addOption(
+                name: 'major-only',
+                shortcut: 'M',
+                mode: InputOption::VALUE_NONE,
+                description: 'Only shows packages that have major SemVer-compatible updates.',
+            )
+            ->addOption(
+                name: 'minor-only',
+                shortcut: 'm',
+                mode: InputOption::VALUE_NONE,
+                description: 'Only shows packages that have minor SemVer-compatible updates.',
+            )
+            ->addOption(
+                name: 'patch-only',
+                shortcut: 'p',
+                mode: InputOption::VALUE_NONE,
+                description: 'Only shows packages that have patch-level SemVer-compatible updates.',
+            )
```

Comment:
I would be ok with NO including the 5 options above. You did an amazing job replicating Composer - seriously. But I'm not sure we need ALL the same options - and it might be better to have a simpler version, at least at first until the need proves itself.